lib/plugins/exchange/consts.js

```diff
@@ -0,0 +1,11 @@
+
+const ORDER_TYPES = {
+  MARKET: 'market',
+  LIMIT: 'limit'
+}
+const DECIMAL_PRECISION = {
+  PRICE: 2,
+  DEFAULT_AMOUNT: 8
+}

```

Comment:
This looks to be used only in `exchange/ccxt.js. You just add these consts there

```
const DEFAULT_PRICE_PRECISION = 2
const DEFAULT_AMOUNT_PRECISION = 8
```